pkg/apis/config/features.go

```diff
@@ -36,15 +36,18 @@ const (
 
 func defaultFeaturesConfig() *Features {
 	return &Features{
-		MultiContainer: Disabled,
+		MultiContainer:     Disabled,
+		KubernetesFieldRef: Disabled,
 	}
 }
 
 // NewFeaturesConfigFromMap creates a Features from the supplied Map
 func NewFeaturesConfigFromMap(data map[string]string) (*Features, error) {
 	nc := defaultFeaturesConfig()
 
-	if err := cm.Parse(data, asFlag("multi-container", &nc.MultiContainer)); err != nil {
+	if err := cm.Parse(data,
+		asFlag("multi-container", &nc.MultiContainer),
+		asFlag("kubernetes/field-ref", &nc.KubernetesFieldRef)); err != nil {
```

Comment:
note: there's a discussion about the format of these fields

https://knative.slack.com/archives/CU9DGL4FM/p1591710914146300